### PR TITLE
Fix swipe navigation conference issue in iOS

### PR DESCRIPTION
**Pull request by Bindu-Maharudrappa**

### CODE REVIEW

File: test_app/lib/screens/participant_screen/conference_controls.dart [hunk 1]
```diff
@@ -4,18 +4,20 @@ import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
...
+  const ConferenceControls({Key? key, required this.conference, required this.updateCloseSessionFlag}) : super(key: key);
 
   @override
   State<ConferenceControls> createState() => _ConferenceControlsState();
 }
 
 class _ConferenceControlsState extends State<ConferenceControls> {
   final _dolbyioCommsSdkFlutterPlugin = DolbyioCommsSdk.instance;
-  bool isMicOff = false, isVideoOff = false;
+  bool isMicOff = false, isVideoOff = false; // closeSessionOnDeactivate = false;
```
Comment (graduad) [inline test_app/lib/screens/participant_screen/conference_controls.dart]: Commented out code

File: test_app/lib/screens/participant_screen/conference_controls.dart [hunk 2]
```diff
@@ -138,25 +138,15 @@ class _ConferenceControlsState extends State<ConferenceControls> {
...
-        });
-      }
+    if (closeSession) {
+      widget.updateCloseSessionFlag(true);
+      Navigator.of(context).popUntil(ModalRoute.withName("LoginScreen"));
+    } else {
+      widget.updateCloseSessionFlag(false);
+
+      Navigator.of(context)
+          .popUntil(ModalRoute.withName("JoinConferenceScreen"));
```
Comment (graduad) [inline test_app/lib/screens/participant_screen/conference_controls.dart:162]: Now this contains so little code that we can move it to the button handlers.

**Review by graduad — CHANGES_REQUESTED** [review]

File: test_app/lib/screens/participant_screen/conference_controls.dart (hunk 2 above)
Comment (Bindu-Maharudrappa) [reply test_app/lib/screens/participant_screen/conference_controls.dart]: I updated it.

File: test_app/lib/screens/participant_screen/conference_controls.dart (hunk 1 above)
Comment (Bindu-Maharudrappa) [reply test_app/lib/screens/participant_screen/conference_controls.dart]: It’s removed now.

**Review by graduad — APPROVED** [review]

**Pull request merged**